release-notes/opensearch.release-notes-2.1.0.md

```diff
@@ -0,0 +1,77 @@
+## 2022-07-05 Version 2.1.0 Release Notes
+
+
+### Breaking Changes in 2.1.0
+
+#### Remove Mapping types
+
+* [Type removal] Ignore _type field in bulk request ([#3505](https://github.com/opensearch-project/opensearch/pull/3505))
```

Comment:
This commit was reverted as part of https://github.com/opensearch-project/OpenSearch/pull/3560. We can remove this commit.